src/gas-hub.js

```diff
@@ -389,7 +389,7 @@ function showDiff(code, type) {
         return true;
       }
       for (let i = 0; i < config.ignorePattern.length; i++) {
-        let p = new RegExp(svconfig.ignorePattern[i]);
+        let p = new RegExp(config.ignorePattern[i]);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/0223d49b4d26b7614407243d511f2ef00f47776a/src/gas-hub.js#L392"></span>‘p’ is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>